### ISSUE TITLE
Sum is_convergent returns incorrect result

### ISSUE THREAD

**Issue by proy87** [sympy/sympy#19836]
`Sum(4/(n + 2) - 5/(n + 1) + 1/n,(n,7,oo)).is_convergent()` returns `False`, but this series is convergent.

**Comment by maurogaravello** [comment]
In master, the result is `True`

**Comment by oscarbenjamin** [comment]
This was fixed in b777e4693084cf731595594d5389c5cd7901fbb8 from #19546. This issue could be closed if a test for this case was added as well.

**Comment by abhaydhiman** [comment]
```
def test_issue_19836():
    n = Symbol('n')
    assert Sum(4/(n+2) - 5/(n+1) + 1/n, (n, 7, oo)).is_convergent() == True
```
Is it good to go?
